src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TribunalOfficeServiceTest.java

```diff
@@ -47,8 +47,8 @@ public class TribunalOfficeServiceTest {
             { TribunalOffice.DUNDEE.getOfficeName(), "DD1 4QB" },
             { TribunalOffice.DUNDEE.getOfficeName(), "DD1 4QB" },
             { TribunalOffice.EDINBURGH.getOfficeName(), "EH3 7HF" },
-            { TribunalOffice.EDINBURGH.getOfficeName(), "EH3 7HF" },
-            { TribunalOffice.EDINBURGH.getOfficeName(), "EH3 7HF" }
+            { "Unassigned", "" },
```

Comment:
Just a minor thing, may be use Unassigned constant here?

Reply:
Added constant to test